src/v0/destinations/google_adwords_offline_conversions/transform.js

```diff
@@ -59,7 +60,20 @@ const getConversions = (message, metadata, { Config }, event, conversionType) =>
     payload = constructPayload(message, trackCallConversionsMapping);
     endpoint = CALL_CONVERSION.replace(':customerId', filteredCustomerId);
   }
-
+  const consentObject = populateConsentForGoogleDestinations(message, conversionType);
+  if (Object.keys(consentObject)?.length > 0) {
+    if (payload?.conversions?.length > 0) {
+      if (conversionType === 'click' || conversionType === 'call') {
+        payload.conversions[0].consent = consentObject;
```

Comment:
Is it enough to populate the consent data only in the first item?

Reply:
yes, according to the implememtation in config json, we can only expect one single object